Source/Core/CommandLineOptions.cs

```diff
@@ -2416,10 +2398,8 @@ be n (default 500)
                 only for monomorphic programs.
   /reflectAdd   In the VC, generate an auxiliary symbol, elsewhere defined
                 to be +, instead of +.
-  /prune:<n>
-                0 (default) - none
-                1 - automatic pruning
-                2 - aggressive pruning. Requires binding axioms to functions and constants using 'uses'
+  /prune
```

Comment:
We should probably summarize here what pruning does.

Reply:
Also, it seems that you did not add the print pruned options to help.

Reply:
I'm not sure how to add a useful summary. I can describe the use-case in a few sentences but the full explanation is 50 lines and I think that's too much for the generic help.

`/printPruned` is described in the help right?

```
/printPruned:<file>
                After pruning, print the Boogie program to the specified file.
```

Reply:
How about simply:
```
/prune
     Remove any top-level Boogie declarations that are not accessible by the implementation that is about to be verified.
```
Even just this I think gives us a good hint for what is going on here.

Reply:
That's in there now